### DISCUSSION TITLE
Is there support for Nuxt v2 / Vue v2?

### DISCUSSION THREAD

**Discussion by p19ky** [graphieros/vue-data-ui#135] [Q&A] [answered]
Is there support for Nuxt v2 / Vue v2?

**Comment by graphieros** [comment, marked as answer]
@p19ky unfortunately not, sorry:)

**Reply by p19ky** [reply]
Oh.. Such a cool project tho! Thanks for the answer!

**Reply by graphieros** [reply]
Thank you, that's very kind^^
